setup.py

```diff
@@ -70,8 +70,8 @@ def to_uri(filename):
         'Operating System :: OS Independent',
         'Topic :: Scientific/Engineering :: Medical Science Apps.'
         ],
-    install_requires=['xsdata==21.12', 'numpy', 'h5py>=2.3'],
-    setup_requires=['nose>=1.0', 'xsdata[cli]==21.12', 'jinja2 >= 2.11'],
+    install_requires=['xsdata==22.2', 'numpy', 'h5py==3.6.0'],
```

Comment:
I don't think we should be pinning h5py to a specific version and not this high. I don't really see a benefit of bumping this since the backwards compat is nice. 

On xsdata, we had a prior issue that lead us to pin this #45. I would propose if we change it here that we go for 22.*.

Reply:
Good point, leaving h5py>2.3 doesn't prevent one from using a later software version, which is installed by default if not otherwise available.